documentation/modules/configuring/con-config-mirrormaker2-sync-acls.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='con-config-mirrormaker2-sync-acls-{context}']
+= Synchronizing ACL rules
+
+[role="_abstract"]
+MirrorMaker 2 can synchronize topic ACL rules from source to target clusters. 
+Enable this feature by configuring the source connector.
+
+With `simple` authorization:
+
+* ACL rules apply to both local and remote topics
```

Comment:
Do we define "local" and "remote" topics somewhere? Also on the next line we've used "source" and "remote"